build.gradle

```diff
@@ -16,6 +16,7 @@ allprojects {
     group = 'bio.terra'
     version = System.getenv('SERVICE_VERSION') != null ? System.getenv('SERVICE_VERSION') : '0.5.0-SNAPSHOT'
     sourceCompatibility = JavaVersion.VERSION_11
+    ext['log4j2.version'] = '2.16.0' // until Spring Boot 2.6.2
```

Comment:
what does until spring boot 2.6.2 means?

Reply:
We currently pull in the log4j-api jars as transitive dependencies of Spring (and possibly other dependencies as well). Spring has said they'll upgrade their dependency version in the 2.6.2 release (expected late December 2021, https://spring.io/blog/2021/12/10/log4j2-vulnerability-and-spring-boot), so at that point we may not need to explicitly override their version here.